themes/custom_tmpl/GETTING_STARTED.md

```diff
@@ -0,0 +1,37 @@
+# Configuration
```

Comment:
This documentation is very helpful. However, before we merge, I think we should talk about our documentation strategy moving forward. Some of the information here overlaps with information in the wiki. We should probably try to avoid redundancy, or else we have more things that can potentially fall out of date and get ignored. Should we link from the wiki to here, or from here to the wiki? Should we think about best practices for what goes in the wiki (perhaps general information that changes infrequently) and what goes in GitHub (perhaps more detailed information that may be volatile -- this could give us a form of version-specific documentation).

Bottom line: we seem to be setting some sort of precedent here, so let's make sure we have a strategy before we go ahead with it. I like what you've done, though!